web/lib/HabitatTransferBox.js

```diff
@@ -257,6 +257,21 @@ habitat-transfer-box .dropdown::after {
 </div>
 `;
 
+window.addEventListener('DOMContentLoaded', function (e) {
```

Comment:
Like above, move this into the class on `connectedCallback` and start listening on the proper events.